app/assets/stylesheets/layout_old/right-column.scss

```diff
@@ -78,7 +78,7 @@
     position: relative;
     display: inline-block;
     float: right;
-    max-width: 206px;
+    width: 206px;
```

Comment:
If you look at the sample pages from the story: 

- https://www.chapman.edu/students/academic-resources/tutoring-center/tutoring/index.aspx
- https://www.chapman.edu/crean/academic-programs/graduate-programs/ms-athletic-training/index.aspx

You can see the content of the right column text is pretty small. Since this setting was max-width instead of width, and the content itself didn't take up 206px, it would make the whole column smaller.